### DISCUSSION TITLE
Welcome to WRF_IMDAAv2 Discussions!

### DISCUSSION THREAD

**Discussion by vivekhazra** [NCMRWF/WRF_IMDAAv2#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by athulya-mrfp** [comment]
I have been running WRF using ERA5 as input and boundary files. Now I am trying to run it using the NGFS 6 hourly data from NCMRWF. I ran WPS using Vtable.GFS and successfully got met_ files. When I try to run real i am getting the error "lag_soil_layers read from met_em file is  1
Max map factor in domain 1 =  1.13. Scale the dt in the model accordingly.
ERROR in psfc: flag_psfc =  1, flag_soilhgt =  0, flag_slp =  0, sfcp_to_sfcp = F
-------------- FATAL CALLED ---------------
FATAL CALLED FROM FILE:  <stdin>  LINE:    1404
not enough info for a p sfc computation
". Can you help me on this?
